engine/jobmaster/dm/checkpoint/agent_test.go

```diff
@@ -34,15 +34,17 @@ import (
 
 func TestTableName(t *testing.T) {
 	t.Parallel()
-	jobCfg := &config.JobCfg{Name: "test", MetaSchema: "meta"}
-	require.Equal(t, loadTableName(jobCfg), "`meta`.`test_lightning_checkpoint_list`")
-	require.Equal(t, syncTableName(jobCfg), "`meta`.`test_syncer_checkpoint`")
-	require.Equal(t, shardMetaName(jobCfg), "`meta`.`test_syncer_sharding_meta`")
-	require.Equal(t, onlineDDLName(jobCfg), "`meta`.`test_onlineddl`")
+	jobID := "test"
+	jobCfg := &config.JobCfg{MetaSchema: "meta"}
+	require.Equal(t, loadTableName(jobID, jobCfg), "`meta`.`test_lightning_checkpoint_list`")
```

Comment:
is Job ID an UUID? please check if the table name is too long

Reply:
Job ID  will be an increasing num in dataflow service.
I think it's better to move this rule to the precheck if it may cause the whole job fail. @GMHDBJD

Reply:
Move this check to the precheck phase if possible.

Reply:
yes, https://github.com/pingcap/tiflow/issues/6785

Reply:
[fd7f702](https://github.com/pingcap/tiflow/pull/6919/commits/fd7f70211830ea70d8e45ed988ff8869b437800e)